lib/dl_api_lib/pyproject.toml

```diff
@@ -16,36 +16,39 @@ flask-marshmallow = ">=1.1.0"
 flask-restx = ">=1.1.0"
 marshmallow = ">=3.19.0"
 marshmallow-oneofschema = ">=3.0.1"
+more-itertools = ">=9.1.0"
 python = ">=3.10, <3.13"
 werkzeug = ">=2.2.3"
 statcommons = {path = "../statcommons"}
-datalens-api-commons = {path = "../dl_api_commons"}
-datalens-utils = {path = "../dl_utils"}
-datalens-constants = {path = "../dl_constants"}
-datalens-compeng-pg = {path = "../dl_compeng_pg"}
-datalens-i18n = {path = "../dl_i18n"}
-datalens-formula = {path = "../dl_formula"}
-datalens-configs = {path = "../dl_configs"}
-datalens-api-connector = {path = "../dl_api_connector"}
-datalens-core = {path = "../dl_core"}
-datalens-app-tools = {path = "../dl_app_tools"}
-datalens-query-processing = {path = "../dl_query_processing"}
-datalens-model-tools = {path = "../dl_model_tools"}
-datalens-task-processor = {path = "../dl_task_processor"}
-datalens-dashsql = {path = "../../lib/dl_dashsql"}
-datalens-pivot = {path = "../dl_pivot"}
-datalens-pivot-pandas = {path = "../dl_pivot_pandas"}
-datalens-cache-engine = {path = "../dl_cache_engine"}
-datalens-rls = {path = "../dl_rls"}
+dl-api-commons = {path = "../dl_api_commons"}
+dl-utils = {path = "../dl_utils"}
+dl-constants = {path = "../dl_constants"}
+dl-compeng-pg = {path = "../dl_compeng_pg"}
+dl-i18n = {path = "../dl_i18n"}
+dl-formula = {path = "../dl_formula"}
+dl-configs = {path = "../dl_configs"}
+dl-api-connector = {path = "../dl_api_connector"}
+dl-core = {path = "../dl_core"}
+dl-app-tools = {path = "../dl_app_tools"}
+dl-query-processing = {path = "../dl_query_processing"}
+dl-model-tools = {path = "../dl_model_tools"}
+dl-task-processor = {path = "../dl_task_processor"}
+dl-dashsql = {path = "../../lib/dl_dashsql"}
+dl-pivot = {path = "../dl_pivot"}
+dl-pivot-pandas = {path = "../dl_pivot_pandas"}
+dl-cache-engine = {path = "../dl_cache_engine"}
+dynamic-enum = {path = "../dynamic_enum"}
+typing-extensions = "==4.9.0"
```

Comment:
let's use >=

Reply:
are u sure? it is typing-extensions = "==4.9.0" in metapkg